.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+### Resolves:
+
+- What this PR resolves
+
+### Changes:
+
+- Bullet list of changes made
+
+### References:
+
```

Comment:
Another section `## Safety` I'd include in which we can describe what steps were taken to ensure that we're not introducing any breaking changes (_eg. testing, lint passes, prettier done, etc._).